api/settings.py

```diff
@@ -76,3 +76,6 @@
 
 # DB
 DATABASE_URL='postgresql://localhost/maap_dev'
+
+# OGC API - Features
+OGCAPI_FEATURES_ENDPOINT = "https://8k07ljl96e.execute-api.us-east-1.amazonaws.com"
```

Comment:
setting this should be documented somewhere, but I don't know where it's supposed to go

Reply:
https://docs.google.com/spreadsheets/d/1Lg4uFH2rM_avTcgWQpeVwS80l_J3rBB_ZSqGTBvyqhU/edit#gid=0
However this is why we started keeping info in SSM parameters, I wonder if we should get it from there.

Reply:
we also have (at least) TiTiler and the 3d tiles server also. If we want to start using SSM for service discovery across services, we should probably do this as a separate piece of work. The only use I've seen so far is for self-discovery of URL within a service.

Reply:
I think we will create a discovery service which uses either service tags or SSM to return an API endpoint. But haven't designed this yet. This approach is fine for now and I have started a ticket in trello for the discovery service since I realize this is become burdensome to manage these endpoints.